packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +44,24 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.title) {
+      if (frameOptions.iframeAttrs == null) {
```

Comment:
Could we use `===` instead of `==`? Same for L57

Reply:
It's different. I am using `==` to check for `null` or `undefined`. I can adjust to include a check for `undefined` as well to make it clear.

Reply:
I updated the check here:
https://github.com/cerner/terra-framework/pull/1776/files#diff-77b1c1b39ad95249bd3956ef509a0995b4687f8c94ba87a3110f3e6b1ce8ccfeR51-R59